Source/Scene/Scene.js

```diff
@@ -811,6 +815,17 @@ define([
             }
         },
 
+
+        /**
+         * @memberof Scene.prototype
+         * @type {ComputeEngine}
+         */
+        computeEngine : {
```

Comment:
Remove this getter.  It is OK for functions in this file that take `scene` as a parameter to access `_computeEngine` directly.  Think of those function as private member functions.